src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasTableUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A class containing convenience methods used in the strand bias annotation calculations
+ */
+public class StrandBiasTableUtils {
+
+    private final static Logger logger = LogManager.getLogger(StrandBiasTableUtils.class);
+
+    //For now this is only for 2x2 contingency tables
+    protected static final int ARRAY_DIM = 2;
```

Comment:
why are those protected and not private? I find those two consts kind of goofy anyway, given that the getContingencyArray clearly hardwires everything to 2x2